cluster/gce/util.sh

```diff
@@ -1868,7 +1868,7 @@ function get-master-replicas-count() {
 # Prints regexp for full master machine name. In a cluster with replicated master,
 # VM names may either be MASTER_NAME or MASTER_NAME with a suffix for a replica.
 function get-replica-name-regexp() {
-  echo "${MASTER_NAME}(-...)?"
+  echo "^${MASTER_NAME}(-...)?"
```

Comment:
why not use `kubernetes-staging`? :)

Reply:
Good point :) Had I known I was gonna run into this issue I would have totally done that! I'm just hoping this will help someone else down the line.

Reply:
I'm not sure I understand this change. What's the motivation of it?

Reply:
I wrote a little bit about it in the PR description above, but basically, I'm running a production cluster and staging cluster and I have GCE instances named `kubernetes-master-001`, `kubernetes-master-002`, etc and more GCE instances named `staging-kubernetes-master-001`, `staging-kubernetes-master-002`, etc. 

When running `kube-up.sh` to create another production master, the `INITIAL_ETCD_CLUSTER` contains the production and staging instances when it should only be production instances. As a result, etcd fails to startup with an errors like `member count unequal` -> the api server fails to startup -> nothing else works and the newly created master is broken.

This change makes sure that the `INITIAL_ETCD_CLUSTER` only contains the production master instances.